tests/components/homematic/test_notify.py

```diff
@@ -61,6 +61,7 @@ async def test_setup_without_optional(hass: HomeAssistant) -> None:
 async def test_bad_config(hass: HomeAssistant) -> None:
     """Test invalid configuration."""
     config = {notify_comp.DOMAIN: {"name": "test", "platform": "homematic"}}
+    await async_setup_component(hass, "repairs", {})
```

Comment:
This seems like an antipattern. I'd change the assert_setup_component call instead to not count the platforms set up.